pkg/services/ngalert/api/tooling/definitions/admin.go

```diff
@@ -76,13 +76,11 @@ const (
 
 // swagger:model
```

Comment:
Since we touched some of the Swagger models, we'll need to regenerate the OpenAPI docs to uptake these changes. Check the README and makefile in this directory: https://github.com/grafana/grafana/tree/main/pkg/services/ngalert/api/tooling

Reply:
Thanks